lib/hotel.rb

```diff
@@ -0,0 +1,92 @@
+require_relative 'room'
+require_relative 'reservation'
+
+module HotelSystem
+  class Hotel
+    attr_reader :rooms, :reservations
+    attr_accessor :amount_of_rooms
+    
+    def initialize(amount_of_rooms)
+      @amount_of_rooms = amount_of_rooms
+      @rooms = []
+      @reservations = []
+    end
+    
+    def create_rooms
+      cost = 200
+      room_number = 0
+      @amount_of_rooms.times do
+        room_number += 1 
+        room = HotelSystem::Room.new(room_number: room_number, cost: cost)
+        rooms.push(room)
+      end
+      
+    end
+    
+    def list_rooms
+      create_rooms
+      rooms
```

Comment:
While the last line of a method is returned, it's best practice to explicitly return: `return rooms`